src/licensedcode/data/licenses/here-proprietary.yml

```diff
@@ -0,0 +1,7 @@
+key: here-proprietary
+name: HERE Proprietary License
+short_name: HERE Proprietary
+category: Proprietary
+owner: HERE Global B.V.
```

Comment:
@pombredanne Does this line need to be adjusted to contain "and its affiliate(s)" after #965 has been addressed?